tests/test_tasks.py

```diff
@@ -75,3 +76,54 @@ def test_update_course_skills_task_with_no_course_found(self, get_course_skills_
 
         self.assertEqual(skill.count(), 0)
         self.assertEqual(course_skill.count(), 0)
+
+    @mock.patch('taxonomy.tasks.get_program_metadata_provider')
+    @mock.patch('taxonomy.tasks.utils.EMSISkillsApiClient.get_product_skills')
+    def test_update_program_skills_task(self, get_program_skills_mock, mock_get_provider):
+        """
+        Verify that `update_program_skills` task work as expected.
+        """
+        get_program_skills_mock.return_value = self.skills_emsi_client_response
+        mock_get_provider.return_value = DiscoveryProgramMetadataProvider([self.program])
+
+        # verify that no `Skill` and `ProgramSkill` records exist before executing the task
+        skill = Skill.objects.all()
+        program_skill = ProgramSkill.objects.all()
+        self.assertEqual(skill.count(), 0)
+        self.assertEqual(program_skill.count(), 0)
```

Comment:
these lines can be converted into a util method because they are in use across various unit tests.